tests/src/test/scala/com/nvidia/spark/rapids/RegularExpressionTranspilerSuite.scala

```diff
@@ -308,7 +319,10 @@ class RegularExpressionTranspilerSuite extends FunSuite with Arm {
     assert(transpiled === expected)
   }
 
-  test("transpile complex regex 2") {
+  // TODO
+  // this was a test for transpiling but we did not ever try to run the
+  // resulting regex to see if it produced valid results
```

Comment:
Is there a followup issue for this?  Would be nice to link it here.

Reply:
I filed https://github.com/NVIDIA/spark-rapids/issues/5656 and linked to it